src/commands/exile_commands.py

```diff
@@ -53,10 +55,16 @@ async def exile(
                 ephemeral=True,
             )
             return
-
+        nicer_duration_str = nicer_duration(duration)
```

Comment:
don't think we need a new variable for this. we can just convert the duration on line 65